app/src/main/java/com/teamltt/carcare/fragment/MyDynamicDataAdapter.java

```diff
@@ -0,0 +1,77 @@
+///*
```

Comment:
This whole class is commented out. Is this because the adapter is commented out in the HomeActivity?